
From: Deepak Saxena <dsaxena@plexity.net>

Spaces in driver names show up as spaces in sysfs.  Annoying.  I went ahead
and changed ones that don't have spaces to use ${NAME}_IDE so they are all
consistent.

Signed-off-by: Deepak Saxena <dsaxena@plexity.net>
Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/drivers/ide/pci/aec62xx.c      |    2 +-
 25-akpm/drivers/ide/pci/alim15x3.c     |    2 +-
 25-akpm/drivers/ide/pci/amd74xx.c      |    2 +-
 25-akpm/drivers/ide/pci/atiixp.c       |    2 +-
 25-akpm/drivers/ide/pci/cmd64x.c       |    2 +-
 25-akpm/drivers/ide/pci/cs5520.c       |    2 +-
 25-akpm/drivers/ide/pci/cy82c693.c     |    2 +-
 25-akpm/drivers/ide/pci/generic.c      |    2 +-
 25-akpm/drivers/ide/pci/hpt34x.c       |    2 +-
 25-akpm/drivers/ide/pci/hpt366.c       |    2 +-
 25-akpm/drivers/ide/pci/it8172.c       |    2 +-
 25-akpm/drivers/ide/pci/ns87415.c      |    2 +-
 25-akpm/drivers/ide/pci/opti621.c      |    2 +-
 25-akpm/drivers/ide/pci/pdc202xx_new.c |    2 +-
 25-akpm/drivers/ide/pci/pdc202xx_old.c |    2 +-
 25-akpm/drivers/ide/pci/piix.c         |    2 +-
 25-akpm/drivers/ide/pci/rz1000.c       |    2 +-
 25-akpm/drivers/ide/pci/sc1200.c       |    2 +-
 25-akpm/drivers/ide/pci/serverworks.c  |    2 +-
 25-akpm/drivers/ide/pci/sgiioc4.c      |    2 +-
 25-akpm/drivers/ide/pci/siimage.c      |    2 +-
 25-akpm/drivers/ide/pci/sis5513.c      |    2 +-
 25-akpm/drivers/ide/pci/sl82c105.c     |    2 +-
 25-akpm/drivers/ide/pci/slc90e66.c     |    2 +-
 25-akpm/drivers/ide/pci/triflex.c      |    2 +-
 25-akpm/drivers/ide/pci/trm290.c       |    2 +-
 25-akpm/drivers/ide/pci/via82cxxx.c    |    2 +-
 27 files changed, 27 insertions(+), 27 deletions(-)

diff -puN drivers/ide/pci/aec62xx.c~remove-spaces-from-pci-ide-pci_drivername-field drivers/ide/pci/aec62xx.c
--- 25/drivers/ide/pci/aec62xx.c~remove-spaces-from-pci-ide-pci_drivername-field	Mon Aug  9 17:39:01 2004
+++ 25-akpm/drivers/ide/pci/aec62xx.c	Mon Aug  9 17:39:01 2004
@@ -540,7 +540,7 @@ static struct pci_device_id aec62xx_pci_
 MODULE_DEVICE_TABLE(pci, aec62xx_pci_tbl);
 
 static struct pci_driver driver = {
-	.name		= "AEC62xx IDE",
+	.name		= "AEC62xx_IDE",
 	.id_table	= aec62xx_pci_tbl,
 	.probe		= aec62xx_init_one,
 };
diff -puN drivers/ide/pci/alim15x3.c~remove-spaces-from-pci-ide-pci_drivername-field drivers/ide/pci/alim15x3.c
--- 25/drivers/ide/pci/alim15x3.c~remove-spaces-from-pci-ide-pci_drivername-field	Mon Aug  9 17:39:01 2004
+++ 25-akpm/drivers/ide/pci/alim15x3.c	Mon Aug  9 17:39:01 2004
@@ -893,7 +893,7 @@ static struct pci_device_id alim15x3_pci
 MODULE_DEVICE_TABLE(pci, alim15x3_pci_tbl);
 
 static struct pci_driver driver = {
-	.name		= "ALI15x3 IDE",
+	.name		= "ALI15x3_IDE",
 	.id_table	= alim15x3_pci_tbl,
 	.probe		= alim15x3_init_one,
 };
diff -puN drivers/ide/pci/amd74xx.c~remove-spaces-from-pci-ide-pci_drivername-field drivers/ide/pci/amd74xx.c
--- 25/drivers/ide/pci/amd74xx.c~remove-spaces-from-pci-ide-pci_drivername-field	Mon Aug  9 17:39:01 2004
+++ 25-akpm/drivers/ide/pci/amd74xx.c	Mon Aug  9 17:39:01 2004
@@ -520,7 +520,7 @@ static struct pci_device_id amd74xx_pci_
 MODULE_DEVICE_TABLE(pci, amd74xx_pci_tbl);
 
 static struct pci_driver driver = {
-	.name		= "AMD IDE",
+	.name		= "AMD_IDE",
 	.id_table	= amd74xx_pci_tbl,
 	.probe		= amd74xx_probe,
 };
diff -puN drivers/ide/pci/atiixp.c~remove-spaces-from-pci-ide-pci_drivername-field drivers/ide/pci/atiixp.c
--- 25/drivers/ide/pci/atiixp.c~remove-spaces-from-pci-ide-pci_drivername-field	Mon Aug  9 17:39:01 2004
+++ 25-akpm/drivers/ide/pci/atiixp.c	Mon Aug  9 17:39:01 2004
@@ -490,7 +490,7 @@ static struct pci_device_id atiixp_pci_t
 MODULE_DEVICE_TABLE(pci, atiixp_pci_tbl);
 
 static struct pci_driver driver = {
-	.name		= "ATIIXP IDE",
+	.name		= "ATIIXP_IDE",
 	.id_table	= atiixp_pci_tbl,
 	.probe		= atiixp_init_one,
 };
diff -puN drivers/ide/pci/cmd64x.c~remove-spaces-from-pci-ide-pci_drivername-field drivers/ide/pci/cmd64x.c
--- 25/drivers/ide/pci/cmd64x.c~remove-spaces-from-pci-ide-pci_drivername-field	Mon Aug  9 17:39:01 2004
+++ 25-akpm/drivers/ide/pci/cmd64x.c	Mon Aug  9 17:39:01 2004
@@ -760,7 +760,7 @@ static struct pci_device_id cmd64x_pci_t
 MODULE_DEVICE_TABLE(pci, cmd64x_pci_tbl);
 
 static struct pci_driver driver = {
-	.name		= "CMD64x IDE",
+	.name		= "CMD64x_IDE",
 	.id_table	= cmd64x_pci_tbl,
 	.probe		= cmd64x_init_one,
 };
diff -puN drivers/ide/pci/cs5520.c~remove-spaces-from-pci-ide-pci_drivername-field drivers/ide/pci/cs5520.c
--- 25/drivers/ide/pci/cs5520.c~remove-spaces-from-pci-ide-pci_drivername-field	Mon Aug  9 17:39:01 2004
+++ 25-akpm/drivers/ide/pci/cs5520.c	Mon Aug  9 17:39:01 2004
@@ -319,7 +319,7 @@ static struct pci_device_id cs5520_pci_t
 MODULE_DEVICE_TABLE(pci, cs5520_pci_tbl);
 
 static struct pci_driver driver = {
-	.name		= "CyrixIDE",
+	.name		= "Cyrix_IDE",
 	.id_table	= cs5520_pci_tbl,
 	.probe		= cs5520_init_one,
 };
diff -puN drivers/ide/pci/cy82c693.c~remove-spaces-from-pci-ide-pci_drivername-field drivers/ide/pci/cy82c693.c
--- 25/drivers/ide/pci/cy82c693.c~remove-spaces-from-pci-ide-pci_drivername-field	Mon Aug  9 17:39:01 2004
+++ 25-akpm/drivers/ide/pci/cy82c693.c	Mon Aug  9 17:39:01 2004
@@ -444,7 +444,7 @@ static struct pci_device_id cy82c693_pci
 MODULE_DEVICE_TABLE(pci, cy82c693_pci_tbl);
 
 static struct pci_driver driver = {
-	.name		= "Cypress IDE",
+	.name		= "Cypress_IDE",
 	.id_table	= cy82c693_pci_tbl,
 	.probe		= cy82c693_init_one,
 };
diff -puN drivers/ide/pci/generic.c~remove-spaces-from-pci-ide-pci_drivername-field drivers/ide/pci/generic.c
--- 25/drivers/ide/pci/generic.c~remove-spaces-from-pci-ide-pci_drivername-field	Mon Aug  9 17:39:01 2004
+++ 25-akpm/drivers/ide/pci/generic.c	Mon Aug  9 17:39:01 2004
@@ -138,7 +138,7 @@ static struct pci_device_id generic_pci_
 MODULE_DEVICE_TABLE(pci, generic_pci_tbl);
 
 static struct pci_driver driver = {
-	.name		= "PCI IDE",
+	.name		= "PCI_IDE",
 	.id_table	= generic_pci_tbl,
 	.probe		= generic_init_one,
 };
diff -puN drivers/ide/pci/hpt34x.c~remove-spaces-from-pci-ide-pci_drivername-field drivers/ide/pci/hpt34x.c
--- 25/drivers/ide/pci/hpt34x.c~remove-spaces-from-pci-ide-pci_drivername-field	Mon Aug  9 17:39:01 2004
+++ 25-akpm/drivers/ide/pci/hpt34x.c	Mon Aug  9 17:39:01 2004
@@ -339,7 +339,7 @@ static struct pci_device_id hpt34x_pci_t
 MODULE_DEVICE_TABLE(pci, hpt34x_pci_tbl);
 
 static struct pci_driver driver = {
-	.name		= "HPT34x IDE",
+	.name		= "HPT34x_IDE",
 	.id_table	= hpt34x_pci_tbl,
 	.probe		= hpt34x_init_one,
 };
diff -puN drivers/ide/pci/hpt366.c~remove-spaces-from-pci-ide-pci_drivername-field drivers/ide/pci/hpt366.c
--- 25/drivers/ide/pci/hpt366.c~remove-spaces-from-pci-ide-pci_drivername-field	Mon Aug  9 17:39:01 2004
+++ 25-akpm/drivers/ide/pci/hpt366.c	Mon Aug  9 17:39:01 2004
@@ -1403,7 +1403,7 @@ static struct pci_device_id hpt366_pci_t
 MODULE_DEVICE_TABLE(pci, hpt366_pci_tbl);
 
 static struct pci_driver driver = {
-	.name		= "HPT366 IDE",
+	.name		= "HPT366_IDE",
 	.id_table	= hpt366_pci_tbl,
 	.probe		= hpt366_init_one,
 };
diff -puN drivers/ide/pci/it8172.c~remove-spaces-from-pci-ide-pci_drivername-field drivers/ide/pci/it8172.c
--- 25/drivers/ide/pci/it8172.c~remove-spaces-from-pci-ide-pci_drivername-field	Mon Aug  9 17:39:01 2004
+++ 25-akpm/drivers/ide/pci/it8172.c	Mon Aug  9 17:39:01 2004
@@ -302,7 +302,7 @@ static struct pci_device_id it8172_pci_t
 MODULE_DEVICE_TABLE(pci, it8172_pci_tbl);
 
 static struct pci_driver driver = {
-	.name		= "IT8172IDE",
+	.name		= "IT8172_IDE",
 	.id_table	= it8172_pci_tbl,
 	.probe		= it8172_init_one,
 };
diff -puN drivers/ide/pci/ns87415.c~remove-spaces-from-pci-ide-pci_drivername-field drivers/ide/pci/ns87415.c
--- 25/drivers/ide/pci/ns87415.c~remove-spaces-from-pci-ide-pci_drivername-field	Mon Aug  9 17:39:01 2004
+++ 25-akpm/drivers/ide/pci/ns87415.c	Mon Aug  9 17:39:01 2004
@@ -236,7 +236,7 @@ static struct pci_device_id ns87415_pci_
 MODULE_DEVICE_TABLE(pci, ns87415_pci_tbl);
 
 static struct pci_driver driver = {
-	.name		= "NS87415IDE",
+	.name		= "NS87415_IDE",
 	.id_table	= ns87415_pci_tbl,
 	.probe		= ns87415_init_one,
 };
diff -puN drivers/ide/pci/opti621.c~remove-spaces-from-pci-ide-pci_drivername-field drivers/ide/pci/opti621.c
--- 25/drivers/ide/pci/opti621.c~remove-spaces-from-pci-ide-pci_drivername-field	Mon Aug  9 17:39:01 2004
+++ 25-akpm/drivers/ide/pci/opti621.c	Mon Aug  9 17:39:01 2004
@@ -367,7 +367,7 @@ static struct pci_device_id opti621_pci_
 MODULE_DEVICE_TABLE(pci, opti621_pci_tbl);
 
 static struct pci_driver driver = {
-	.name		= "Opti621 IDE",
+	.name		= "Opti621_IDE",
 	.id_table	= opti621_pci_tbl,
 	.probe		= opti621_init_one,
 };
diff -puN drivers/ide/pci/pdc202xx_new.c~remove-spaces-from-pci-ide-pci_drivername-field drivers/ide/pci/pdc202xx_new.c
--- 25/drivers/ide/pci/pdc202xx_new.c~remove-spaces-from-pci-ide-pci_drivername-field	Mon Aug  9 17:39:01 2004
+++ 25-akpm/drivers/ide/pci/pdc202xx_new.c	Mon Aug  9 17:39:01 2004
@@ -531,7 +531,7 @@ static struct pci_device_id pdc202new_pc
 MODULE_DEVICE_TABLE(pci, pdc202new_pci_tbl);
 
 static struct pci_driver driver = {
-	.name		= "Promise IDE",
+	.name		= "Promise_IDE",
 	.id_table	= pdc202new_pci_tbl,
 	.probe		= pdc202new_init_one,
 };
diff -puN drivers/ide/pci/pdc202xx_old.c~remove-spaces-from-pci-ide-pci_drivername-field drivers/ide/pci/pdc202xx_old.c
--- 25/drivers/ide/pci/pdc202xx_old.c~remove-spaces-from-pci-ide-pci_drivername-field	Mon Aug  9 17:39:01 2004
+++ 25-akpm/drivers/ide/pci/pdc202xx_old.c	Mon Aug  9 17:39:01 2004
@@ -905,7 +905,7 @@ static struct pci_device_id pdc202xx_pci
 MODULE_DEVICE_TABLE(pci, pdc202xx_pci_tbl);
 
 static struct pci_driver driver = {
-	.name		= "Promise Old IDE",
+	.name		= "Promise_Old_IDE",
 	.id_table	= pdc202xx_pci_tbl,
 	.probe		= pdc202xx_init_one,
 };
diff -puN drivers/ide/pci/piix.c~remove-spaces-from-pci-ide-pci_drivername-field drivers/ide/pci/piix.c
--- 25/drivers/ide/pci/piix.c~remove-spaces-from-pci-ide-pci_drivername-field	Mon Aug  9 17:39:01 2004
+++ 25-akpm/drivers/ide/pci/piix.c	Mon Aug  9 17:39:01 2004
@@ -803,7 +803,7 @@ static struct pci_device_id piix_pci_tbl
 MODULE_DEVICE_TABLE(pci, piix_pci_tbl);
 
 static struct pci_driver driver = {
-	.name		= "PIIX IDE",
+	.name		= "PIIX_IDE",
 	.id_table	= piix_pci_tbl,
 	.probe		= piix_init_one,
 };
diff -puN drivers/ide/pci/rz1000.c~remove-spaces-from-pci-ide-pci_drivername-field drivers/ide/pci/rz1000.c
--- 25/drivers/ide/pci/rz1000.c~remove-spaces-from-pci-ide-pci_drivername-field	Mon Aug  9 17:39:01 2004
+++ 25-akpm/drivers/ide/pci/rz1000.c	Mon Aug  9 17:39:01 2004
@@ -74,7 +74,7 @@ static struct pci_device_id rz1000_pci_t
 MODULE_DEVICE_TABLE(pci, rz1000_pci_tbl);
 
 static struct pci_driver driver = {
-	.name		= "RZ1000 IDE",
+	.name		= "RZ1000_IDE",
 	.id_table	= rz1000_pci_tbl,
 	.probe		= rz1000_init_one,
 };
diff -puN drivers/ide/pci/sc1200.c~remove-spaces-from-pci-ide-pci_drivername-field drivers/ide/pci/sc1200.c
--- 25/drivers/ide/pci/sc1200.c~remove-spaces-from-pci-ide-pci_drivername-field	Mon Aug  9 17:39:01 2004
+++ 25-akpm/drivers/ide/pci/sc1200.c	Mon Aug  9 17:39:01 2004
@@ -565,7 +565,7 @@ static struct pci_device_id sc1200_pci_t
 MODULE_DEVICE_TABLE(pci, sc1200_pci_tbl);
 
 static struct pci_driver driver = {
-	.name		= "SC1200 IDE",
+	.name		= "SC1200_IDE",
 	.id_table	= sc1200_pci_tbl,
 	.probe		= sc1200_init_one,
 	.suspend	= sc1200_suspend,
diff -puN drivers/ide/pci/serverworks.c~remove-spaces-from-pci-ide-pci_drivername-field drivers/ide/pci/serverworks.c
--- 25/drivers/ide/pci/serverworks.c~remove-spaces-from-pci-ide-pci_drivername-field	Mon Aug  9 17:39:01 2004
+++ 25-akpm/drivers/ide/pci/serverworks.c	Mon Aug  9 17:39:01 2004
@@ -812,7 +812,7 @@ static struct pci_device_id svwks_pci_tb
 MODULE_DEVICE_TABLE(pci, svwks_pci_tbl);
 
 static struct pci_driver driver = {
-	.name		= "Serverworks IDE",
+	.name		= "Serverworks_IDE",
 	.id_table	= svwks_pci_tbl,
 	.probe		= svwks_init_one,
 #if 0	/* FIXME: implement */
diff -puN drivers/ide/pci/sgiioc4.c~remove-spaces-from-pci-ide-pci_drivername-field drivers/ide/pci/sgiioc4.c
--- 25/drivers/ide/pci/sgiioc4.c~remove-spaces-from-pci-ide-pci_drivername-field	Mon Aug  9 17:39:01 2004
+++ 25-akpm/drivers/ide/pci/sgiioc4.c	Mon Aug  9 17:39:01 2004
@@ -782,7 +782,7 @@ static struct pci_device_id sgiioc4_pci_
 MODULE_DEVICE_TABLE(pci, sgiioc4_pci_tbl);
 
 static struct pci_driver driver = {
-	.name = "SGI-IOC4 IDE",
+	.name = "SGI-IOC4_IDE",
 	.id_table = sgiioc4_pci_tbl,
 	.probe = sgiioc4_init_one,
 };
diff -puN drivers/ide/pci/siimage.c~remove-spaces-from-pci-ide-pci_drivername-field drivers/ide/pci/siimage.c
--- 25/drivers/ide/pci/siimage.c~remove-spaces-from-pci-ide-pci_drivername-field	Mon Aug  9 17:39:01 2004
+++ 25-akpm/drivers/ide/pci/siimage.c	Mon Aug  9 17:39:01 2004
@@ -1137,7 +1137,7 @@ static struct pci_device_id siimage_pci_
 MODULE_DEVICE_TABLE(pci, siimage_pci_tbl);
 
 static struct pci_driver driver = {
-	.name		= "SiI IDE",
+	.name		= "SiI_IDE",
 	.id_table	= siimage_pci_tbl,
 	.probe		= siimage_init_one,
 };
diff -puN drivers/ide/pci/sis5513.c~remove-spaces-from-pci-ide-pci_drivername-field drivers/ide/pci/sis5513.c
--- 25/drivers/ide/pci/sis5513.c~remove-spaces-from-pci-ide-pci_drivername-field	Mon Aug  9 17:39:01 2004
+++ 25-akpm/drivers/ide/pci/sis5513.c	Mon Aug  9 17:39:01 2004
@@ -968,7 +968,7 @@ static struct pci_device_id sis5513_pci_
 MODULE_DEVICE_TABLE(pci, sis5513_pci_tbl);
 
 static struct pci_driver driver = {
-	.name		= "SIS IDE",
+	.name		= "SIS_IDE",
 	.id_table	= sis5513_pci_tbl,
 	.probe		= sis5513_init_one,
 };
diff -puN drivers/ide/pci/sl82c105.c~remove-spaces-from-pci-ide-pci_drivername-field drivers/ide/pci/sl82c105.c
--- 25/drivers/ide/pci/sl82c105.c~remove-spaces-from-pci-ide-pci_drivername-field	Mon Aug  9 17:39:01 2004
+++ 25-akpm/drivers/ide/pci/sl82c105.c	Mon Aug  9 17:39:01 2004
@@ -503,7 +503,7 @@ static struct pci_device_id sl82c105_pci
 MODULE_DEVICE_TABLE(pci, sl82c105_pci_tbl);
 
 static struct pci_driver driver = {
-	.name		= "W82C105 IDE",
+	.name		= "W82C105_IDE",
 	.id_table	= sl82c105_pci_tbl,
 	.probe		= sl82c105_init_one,
 };
diff -puN drivers/ide/pci/slc90e66.c~remove-spaces-from-pci-ide-pci_drivername-field drivers/ide/pci/slc90e66.c
--- 25/drivers/ide/pci/slc90e66.c~remove-spaces-from-pci-ide-pci_drivername-field	Mon Aug  9 17:39:01 2004
+++ 25-akpm/drivers/ide/pci/slc90e66.c	Mon Aug  9 17:39:01 2004
@@ -387,7 +387,7 @@ static struct pci_device_id slc90e66_pci
 MODULE_DEVICE_TABLE(pci, slc90e66_pci_tbl);
 
 static struct pci_driver driver = {
-	.name		= "SLC90e66 IDE",
+	.name		= "SLC90e66_IDE",
 	.id_table	= slc90e66_pci_tbl,
 	.probe		= slc90e66_init_one,
 };
diff -puN drivers/ide/pci/triflex.c~remove-spaces-from-pci-ide-pci_drivername-field drivers/ide/pci/triflex.c
--- 25/drivers/ide/pci/triflex.c~remove-spaces-from-pci-ide-pci_drivername-field	Mon Aug  9 17:39:01 2004
+++ 25-akpm/drivers/ide/pci/triflex.c	Mon Aug  9 17:39:01 2004
@@ -242,7 +242,7 @@ static struct pci_device_id triflex_pci_
 MODULE_DEVICE_TABLE(pci, triflex_pci_tbl);
 
 static struct pci_driver driver = {
-	.name		= "TRIFLEX IDE",
+	.name		= "TRIFLEX_IDE",
 	.id_table	= triflex_pci_tbl,
 	.probe		= triflex_init_one,
 };
diff -puN drivers/ide/pci/trm290.c~remove-spaces-from-pci-ide-pci_drivername-field drivers/ide/pci/trm290.c
--- 25/drivers/ide/pci/trm290.c~remove-spaces-from-pci-ide-pci_drivername-field	Mon Aug  9 17:39:01 2004
+++ 25-akpm/drivers/ide/pci/trm290.c	Mon Aug  9 17:39:01 2004
@@ -408,7 +408,7 @@ static struct pci_device_id trm290_pci_t
 MODULE_DEVICE_TABLE(pci, trm290_pci_tbl);
 
 static struct pci_driver driver = {
-	.name		= "TRM290 IDE",
+	.name		= "TRM290_IDE",
 	.id_table	= trm290_pci_tbl,
 	.probe		= trm290_init_one,
 };
diff -puN drivers/ide/pci/via82cxxx.c~remove-spaces-from-pci-ide-pci_drivername-field drivers/ide/pci/via82cxxx.c
--- 25/drivers/ide/pci/via82cxxx.c~remove-spaces-from-pci-ide-pci_drivername-field	Mon Aug  9 17:39:01 2004
+++ 25-akpm/drivers/ide/pci/via82cxxx.c	Mon Aug  9 17:39:01 2004
@@ -632,7 +632,7 @@ static struct pci_device_id via_pci_tbl[
 MODULE_DEVICE_TABLE(pci, via_pci_tbl);
 
 static struct pci_driver driver = {
-	.name 		= "VIA IDE",
+	.name 		= "VIA_IDE",
 	.id_table 	= via_pci_tbl,
 	.probe 		= via_init_one,
 };
_
